src/main/java/mate/jdbc/service/ManufacturerService.java

```diff
@@ -0,0 +1,17 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Manufacturer;
+
+public interface ManufacturerService {
+    Manufacturer create(Manufacturer manufacturer);
+
+    Optional<Manufacturer> get(Long id);
```

Comment:
<img width="819" alt="image" src="https://user-images.githubusercontent.com/81781479/220949320-4f04e2ee-5d8d-4497-a18d-a4b6468b10b7.png">
 you should not return optional from service layer here